File: code/highest_scoring_word/highest_scoring_word.kt
fun high(str: String) : String {
    var result = 0
    var result_index = 0
    var words = str.split(" ")
    for (i in words.indices) {
        var r = 0
        for (token in words[i]) {
            r += (token.toInt() - 96)
            if (result < r){
                result = r
                result_index = i
            }
        }
    }

    return words[result_index]
}

File: code/highest_scoring_word/highest_scoring_word.py
def high(x):
    result = 0
    result_index = 0
    words = x.split(" ")
    for index, word in enumerate(words):
        r = 0
        for token in word:
            r += (ord(token) - 96)
        if result < r:
            result = r
            result_index = index

    return words[result_index]